service/state/core_access.go

```diff
@@ -157,10 +220,23 @@ func (ca *CoreAccessor) Transfer(
 		return nil, err
 	}
 	coins := types.NewCoins(types.NewCoin(app.BondDenom, amount))
-	msg := banktypes.NewMsgSend(from, to, coins)
+	msg := bank_types.NewMsgSend(from, to, coins)
 	signedTx, err := ca.constructSignedTx(ctx, msg, apptypes.SetGasLimit(gasLim))
 	if err != nil {
 		return nil, err
 	}
 	return ca.SubmitTx(ctx, signedTx)
 }
+
+func sdkErrorToGRPCError(resp sdk_abci.ResponseQuery) error {
```

Comment:
may be it is worth to put such functions in a separate file? `service/state/helpers.go ` or `service/state/utils `